website/content/partials/cli/monitor/flags/log-format.mdx

```diff
@@ -0,0 +1,9 @@
+<a id="monitor-flag-log-format" />
+
+**`-log-format (string: "standard")`**
+
+Format to emit logs.
+Valid formats are "standard", and "json".
+If this option is not specified, "standard" is used.
```

Comment:
```suggestion
```
That sentence isn't needed since it's basically just tell the reader what the default value is, which we've already done :)